lib/main_common.pm

```diff
@@ -458,7 +458,9 @@ sub load_extra_tests() {
         # dependency of git test
         loadtest "console/sshd";
         loadtest "console/zypper_ref";
-        loadtest "console/zypper_info";
+        if (sle_version_at_least('12-SP2') || check_env('DISTRI', 'opensuse')) {
```

Comment:
Hello @mimi1vx,

It does not include test functionality for SLE12SP2+ yet. It is covering something that was not covered until now.
But as I am implementing it step by step, I have first started with SLE12SP2+. If there is a failure happening on other versions it has to be changed on the test module zypper_info. In the future we would like to also test zypper info on SLE12SP2. Something different is, that in the end all this work would serve as a base to test the real new features introduced at SLE12SP2+.

The changes in this PR#2695 should not be necessary after this: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2678